tests/5.0/target_teams_distribute/test_target_teams_distribute_reduction_and.c

```diff
@@ -0,0 +1,88 @@
+//===--- test_target_teams_distribute_reduction_and.c------------------------===//
+//
+// OpenMP API Version 4.5 Nov 2015
+//
+// This test uses the reduction clause on a target teams distribute directive,
+// testing that the variable in the reduction clause is properly reduced using
+// the and operator.
+//
+////===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+
+#define N 1024
+
+int test_and() {
+  char a[N];
+  char result;
+  char host_result;
+      // The below calculation is meant to ensure about half the test arrays will
+      // come out to true under the 'and' operator, and the rest false. For the and
+      // operator, a test array that comes out true requires every entry to be true, 
+      // which is why this margin is so close to 100%.
+  double false_margin = pow(exp(1), log(.5)/N);
+  int errors = 0;
+  int num_teams[N];
+  int warned = 0;
+  srand(1);
+
+  for (int itr_count = 0; itr_count < 16; ++itr_count) {
+    for (int x = 0; x < N; ++x) {
+      if (rand() / (double)(RAND_MAX) < false_margin) {
+	a[x] = 1;
+      } else {
+	a[x] = 0;
+      }
```

Comment:
These lines could be simply changed for `a[x] = rand() / (double)(RAND_MAX) < false_margin`

Reply:
This is done for all tests where it was possible